src/modules/tree.ts

```diff
@@ -488,4 +488,23 @@ export default {
       requiredArgs: 'edition',
     },
   },
+  logs: {
+    description: 'Show logs of an app on the current directory or a specified one',
+    handler: './apps/logs',
+    optionalArgs: ['vendor', 'app'],
+    options: [
+      {
+        description: 'Show logs of all apps of this vendor',
```

Comment:
Seem a bit odd this phrase construction, maybe check with Vladimir a better way to say this?